### ISSUE TITLE
Please consider adding Appimages to your build cycle. 

### ISSUE THREAD

**Issue by llamabyte** [cake-tech/cake_wallet#988]
Several other wallets have an appimage version and once set up it should not add too much to your build cycle, i hope https://appimage.org/

Thanks !

**Comment by aquaspy** [comment]
Agreed on this one. Appimages are distro-agnostic and effortless to use/integrate.
